api_documentation.md

```diff
@@ -0,0 +1,57 @@
+# API Documentation
+
+> Note: all endpoints return JSON
+
+### /api/login
+*method:* **POST**  
+*parameters:* **username**, **sim_id**, **password**  
+*description:* creates a user entry in the database and compares the entered password with the simulation password
+*onsuccess returns:* the user model  
+
+### /api/simulations
+*method:* **GET**  
+*parameters:* **none**  
+*description:* fetches all the simulations
+*onsuccess returns:* list of simulations in the database  
+
+### /api/pages  
+*method:* **POST**  
```

Comment:
Nice work on the API documentation! I'm a bit puzzled why many of the read-only endpoints use POST instead of GET? I understand why the note create/destroy endpoints are POST, but it seems like the read-only endpoints should be using GET.